.github/workflows/ci.yml

```diff
@@ -8,7 +8,6 @@ jobs:
 
     strategy:
       max-parallel: 5
```

Comment:
This should be removed too.

Reply:
This allows you to run each testsuite in parallel, and reduce test time!
But I'll remove it ok.